cmd/influxd/run/config.go

```diff
@@ -69,8 +69,10 @@ func NewConfig() *Config {
 	c.Monitor = monitor.NewConfig()
 	c.Subscriber = subscriber.NewConfig()
 	c.HTTPD = httpd.NewConfig()
+	c.Graphites = []graphite.Config{graphite.NewConfig()}
```

Comment:
Well, they would be added in the disabled mode. I don't think that is an issue. After all, the sample config that ships with the code has both present in the disabled mode.